mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -460,7 +460,7 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 			ARG 2 pos
 			ARG 3 direction
 		METHOD method_26204 getBlock ()Lnet/minecraft/class_2248;
-		METHOD method_26205 getTopMaterialColor (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Lnet/minecraft/class_3620;
+		METHOD method_26205 getTopMapColor (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;)Lnet/minecraft/class_3620;
```

Comment:
Why is `Top` necessary here? As described in the documentation, map colors represent the surface color of the block. This is just a blockstate sensitive getter for the map color, no?